src/main/java/com/gmail/goosius/siegewar/integration/dynmap/DynmapTownyListener.java

```diff
@@ -12,7 +12,10 @@ public class DynmapTownyListener implements Listener {
      *
      * 1. It looks for the %occupier% tag in the popup
      * 2. If the %occupier% tag exists, it replaces it with the occupier name (or blank if there is no occupier)
+     * 
+     * As of SW2.0.0, this method is depreciated, because the town's occupier will always be the same as the town's normal nation
      */
+    @Deprecated
```

Comment:
Does this need to be deprecated or can we just stop using it?

Reply:
- Some servers will have configured their TownyDynmaps to include an "occupier" tag. I believe this is even suggested in the SW install guide
- If we do delete this method, those servers will start getting errors on their dynmap displays (I think just visual errors)
- Therefore since the method works, and it costs us no effort to leave it working for a while, I suggest we mark it depreciated.